### DISCUSSION TITLE
How to request all available (hourly) weather attributes?

### DISCUSSION THREAD

**Discussion by sarthak** [open-meteo/open-meteo#903] [Q&A] [answered]
Hi, is there a short-hand query param that I can pass to request all available hourly weather variables for the selected weather model instead of having to hard-code all variable names in a verbose manner?

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi,
no, there is no shorthand to get all variables. Some weather models offer hundreds of variables if pressure level data is considered.  A request containing all data, would require a lot of resources.
